src/main/java/com/google/sps/python/heatMap.py

```diff
@@ -164,6 +176,6 @@ def load_weighted_cases(population_map):
 
 
 coord_weight_map = load_covid_data()
-dump_file_path = 'heatWeights.txt'
-with open(dump_file_path, 'w') as outfile:
-    json.dump(coord_weight_map, outfile)
+#dump_file_path = 'heatWeights.txt'
+#with open(dump_file_path, 'w') as outfile:
+#    json.dump(coord_weight_map, outfile)
```

Comment:
Don't leave commented code.

If you wanna be able to run this file without overwriting data you could guard this under a flag.